app/controllers/protocols_controller.rb

```diff
@@ -322,8 +322,10 @@ def convert_date_for_save(attrs, date_field)
   def fix_identity
     attrs               = protocol_params
     attrs[:project_roles_attributes].each do |index, project_role|
-      identity = Identity.find_or_create project_role[:identity_id]
-      project_role[:identity_id] = identity.id
+      if project_role[:identity_id].present?
```

Comment:
I noticed this exact code is also in `Dashboard::ProtocolsController.rb`. This change probably needs to happen there as well.

Reply:
Done! Thanks!